### DISCUSSION TITLE
Please upload your pictures here for the wiki gallery

### DISCUSSION THREAD

**Discussion by Matthias32** [ozel/DIY_particle_detector#4] [Show and tell]
![IMG_5474](https://user-images.githubusercontent.com/26323856/86839219-709fa600-c0a1-11ea-848e-94c4675af830.JPEG)
![IMG_5470](https://user-images.githubusercontent.com/26323856/86839222-71d0d300-c0a1-11ea-912d-722952c6ea9f.JPEG)
![IMG_5478](https://user-images.githubusercontent.com/26323856/86839223-72696980-c0a1-11ea-9b91-ab077a0ccf31.JPEG)
![IMG_5479](https://user-images.githubusercontent.com/26323856/86839227-739a9680-c0a1-11ea-8c79-271bf4ea06d0.JPEG)
![IMG_5477](https://user-images.githubusercontent.com/26323856/86839230-739a9680-c0a1-11ea-83c4-500694a403fa.JPEG)

**Comment by ozel** [comment]
Thanks a bunch, @Matthias32! I've added some to the wiki gallery and will embed the rest as examples for a nice radiation window solution in the other wiki pages.  

I am leaving this issue open so that others will hopefully be encouraged to upload their own build pictures as well! :-)

**Comment by ozel** [comment]
adding Malte's build picture from https://www.mikrocontroller.net/topic/495968#6327455 to make it public
![beta](https://user-images.githubusercontent.com/653212/87235154-15d7b880-c3d9-11ea-987e-3fc4223cff35.jpg)

**Comment by Matthias32** [comment]
![IMG_5773](https://user-images.githubusercontent.com/26323856/93909918-a3505680-fd00-11ea-96ba-a92540f5a23c.JPEG)
Mobile version

**Comment by Matthias32** [comment]
Disassembly of foil capacitors
![IMG_5905](https://user-images.githubusercontent.com/26323856/95664714-b5c4ef80-0b4a-11eb-96ae-a4ff08719d7a.JPEG)
![IMG_5906](https://user-images.githubusercontent.com/26323856/95664718-beb5c100-0b4a-11eb-8a62-5f1702e3101b.JPEG)
![IMG_5902](https://user-images.githubusercontent.com/26323856/95664724-d12ffa80-0b4a-11eb-9f2f-ab6e64e1f599.JPEG)

**Reply by ozel** [reply]
thanks Matthias, I've finally added these capacitor pictures to the [Wiki](https://github.com/ozel/DIY_particle_detector/wiki/Diodes#blocking-of-light-using-thin-foils)!

**Reply by SteveF161** [reply]
![image](https://user-images.githubusercontent.com/28903476/116794405-97ade500-aac4-11eb-98d5-fe692137dd8d.jpeg)
![image](https://user-images.githubusercontent.com/28903476/116794438-c926b080-aac4-11eb-90ad-6c362a70b705.jpeg)

Alpha spectrometer and Electron Detector

**Reply by ozel** [reply]
Pictures from a high-school workshop with 12 students in Bonn, December 2020

![8E17B6D1-E1A7-4FCE-8B22-E973F7F2BC6C](https://user-images.githubusercontent.com/653212/122479538-8a44af80-cfcb-11eb-8282-6cea3b60b1fa.jpeg)
![IMG_0297](https://user-images.githubusercontent.com/653212/122479604-a9434180-cfcb-11eb-96f3-43c5ce46ae28.JPG)
![IMG_0287](https://user-images.githubusercontent.com/653212/122479618-b06a4f80-cfcb-11eb-9ec0-69ce2fdb150d.JPG)
![IMG_0288](https://user-images.githubusercontent.com/653212/122479667-c37d1f80-cfcb-11eb-8dc5-562d0e947e7c.JPG)

Another workshop with 16 students in Gmund, May 2021:

![IMG_4263](https://user-images.githubusercontent.com/653212/122480477-35a23400-cfcd-11eb-9758-02e06204f007.JPG)
![IMG_4267](https://user-images.githubusercontent.com/653212/122480502-405cc900-cfcd-11eb-88f1-c20c5428fe92.JPG)
![IMG_4274](https://user-images.githubusercontent.com/653212/122480514-45217d00-cfcd-11eb-9f29-b374033e51ef.JPG)
![IMG_4284](https://user-images.githubusercontent.com/653212/122480572-61251e80-cfcd-11eb-8f13-da42721f2a3b.JPG)

Described in more detail on the FreiesWissen fellow program wiki page: https://github.com/ozel/DIY_particle_detector/wiki/Fellow-FreiesWissen-Project

**Comment by EddieTheHead** [comment]
![particle_detector](https://user-images.githubusercontent.com/7189283/165196660-74331752-29ec-49dc-b34b-624d6cc11292.jpg)

**Comment by seanboyce** [comment]
Moved one capacitor to the other side of the board, so I could place the detector closer to the radiation window. Detector window is a series of 4mm holes drilled directly above the diodes, rather than a square hole. Radiation window covering is copper-clad polyimide film covered with black vinyl tape.
![detector_small](https://github.com/ozel/DIY_particle_detector/assets/24909600/8045f689-2f4e-4cda-820d-c4738481f379)

Externals:

![particle_detector](https://github.com/ozel/DIY_particle_detector/assets/24909600/f33a618e-a3be-4669-a6be-4f3387c6dd5f)

**Reply by ozel** [reply]
nice box, thanks for sharing! 

The placement of the big capacitor always depends on the enclosure. In the instruction PDF and on the wiki it should be pointed out, I think. There is a [small die-cast aluminium enclosure](https://github.com/ozel/DIY_particle_detector/wiki/Enclosures#diecast-aluminum-enclosures) named [Multicomp G102MF](https://octopart.com/search?q=Multicomp+G102MF&currency=EUR&specs=0&in_stock_only=1&sort=median_price_1000&sort-dir=asc), where the board fits exactly inside with the battery on top. When the diodes are facing upside (like the big capacitor in default position) the enclosure plus battery resembles some kind of crude 1.5 cm long collimator 'window'. That was the idea behind the default capacitor/diode placement. But for achieving the highest sensitivity, it is of course desirable to put the diodes as close as possible to the walls of the enclosure as you did.

**Comment by arash258** [comment]
Here are some photos of my build. I was able to see signals on the oscilloscope, and I am still optimizing the circuit. In the alpha spectrometry board, I left a small gap between the BPX61 sensor and the PCB pads.
ARASH A.NIOUSHAN
<img width="1308" height="1241" alt="20260607_164535" src="https://github.com/user-attachments/assets/03fb0e3e-d9ab-4240-b82f-877cd6916ca2" />
<img width="2897" height="1832" alt="20260607_170843" src="https://github.com/user-attachments/assets/5eb724f9-f0dd-4884-b87e-17eff1ae0d80" />
<img width="4000" height="1844" alt="20260607_163749" src="https://github.com/user-attachments/assets/668a11a6-1eb6-4cb9-be4f-e643df8a3f0c" />
<img width="2559" height="1844" alt="20260607_151039" src="https://github.com/user-attachments/assets/63b1ed39-1bf1-45ab-b0e8-58c3d250e963" />
<img width="2663" height="1095" alt="20260607_151131" src="https://github.com/user-attachments/assets/2eb8b44d-b0f8-466e-b76a-bfa94293a52f" />
<img width="3006" height="1326" alt="20260607_151158" src="https://github.com/user-attachments/assets/fe30bfeb-9da9-44c5-84e0-e8b3373b311a" />
<img width="2946" height="1262" alt="20260607_151220" src="https://github.com/user-attachments/assets/4567636d-3596-4791-a615-9262de67c365" />
<img width="1844" height="1132" alt="20260607_173009" src="https://github.com/user-attachments/assets/9eb20582-25b3-4e5f-bf9a-992475b873fb" />
